client/public/index.html

```diff
@@ -48,6 +48,6 @@
 <noscript>
     You need to enable JavaScript to run this app.
 </noscript>
-<div id="root"></div>
+<div class="everything" id="root"></div>
```

Comment:
Lol, might want a better class name than "everything". I always use "content" or sth, but up to you

Reply:
:) renamed it to the-whole-page

Reply:
good enough